### ISSUE TITLE
Regression: Structure panel should group methods under the class name

### ISSUE THREAD

**Issue by jfehrle** [reasonml-editor/reasonml-idea-plugin#268]
plugin version: 0.91-2020.1

Compare this

![image](https://user-images.githubusercontent.com/1253341/93004840-d70cce80-f4ff-11ea-8f2d-cfc8dbeaaa8a.png)

to the first image in https://github.com/reasonml-editor/reasonml-idea-plugin/issues/101.  In the latter, the methods are grouped under the class name.

**Comment by jfehrle** [comment]
```
class tag : text_tag ->
object
  method as_tag : text_tag
  method connect : tag_signals
end
class tag2 : text_tag ->
object
  method as_tag : text_tag
  method connect : tag_signals
end
```

**Comment by giraud** [comment]
it's not a regression because class types where not parsed correctly 😉

**Comment by jfehrle** [comment]
A parsing regression, then.

**Comment by giraud** [comment]
should be fixed in version 0.92
